server/boards/sql/index.ts

```diff
@@ -116,6 +116,20 @@ const updateBoardSettings = `
         settings = $/settings/
     WHERE boards.string_id = $/board_id/`;
 
+const fetchRolesInBoard = `
+		SELECT
+			board_user_roles.user_id as user_id,
```

Comment:
You want to use `users.firebase_id` here and `roles.string_id` as the ids returned by the query. While you join with the internal IDs, these are considered a "sensitive detail" and usually not returned to the client.

For a quick example of why they might be sensitive, because they're generated as sequential entries they reveal:
- How many rows you have in the DB
- The relative order of creation of different entries

This is not always _actually sensitive_ but it's good practice to keep it hidden.